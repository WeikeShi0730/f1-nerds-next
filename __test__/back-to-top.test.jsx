import { render, RenderResult } from "@testing-library/react";
import BackToTop from "../components/back-to-top.component";

let documentBody;
describe("<BackToTop />", () => {
  beforeEach(() => {
    documentBody = render(<BackToTop />);
  });
  it("shows content in <BackToTop />", () => {
    const { baseElement } = documentBody;
    expect(baseElement).toMatchSnapshot();
  });
});
