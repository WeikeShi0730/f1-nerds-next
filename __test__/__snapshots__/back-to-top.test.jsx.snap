// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackToTop /> shows content in <BackToTop /> 1`] = `
<body>
  <div>
    <div
      class="fixed bottom-10 right-10 z-50"
    >
      <button>
        <div
          class="relative w-10 h-10 transition-all duration-200 ease-in-out opacity-0"
        >
          <span
            style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
          >
            <img
              alt="back to top button"
              data-nimg="fill"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
            <noscript />
          </span>
        </div>
      </button>
    </div>
  </div>
</body>
`;
